Makefile

```diff
@@ -1,9 +1,9 @@
 # Current Operator version
-VERSION ?= 2.0.0-snapshot
+VERSION ?= 1.0.0-snapshot
```

Comment:
Product version: 7.11

cc @radtriste @spolti

Reply:
changed to 7.11